modules/utxo-lib/src/bitgo/zcash/ZcashTransactionBuilder.ts

```diff
@@ -21,7 +21,7 @@ export class ZcashTransactionBuilder extends UtxoTransactionBuilder<ZcashTransac
     return new ZcashTransaction(network as ZcashNetwork, tx as ZcashTransaction);
   }
 
-  static fromTransaction(
+  static fromTransactionZcash(
```

Comment:
Method removal. Please record as BREAKING CHANGE in commits.

Reply:
refactoring